emission/storage/modifiable/builtin_model_storage.py

```diff
@@ -15,6 +15,13 @@ def __init__(self, user_id):
         super(BuiltinModelStorage, self).__init__(user_id)
         self.key_query = lambda key: {"metadata.key": key}
         self.user_query = {"user_id": self.user_id} # UUID is mandatory for this version
+        self.current_model = None
```

Comment:
there is not only one model per user
https://github.com/e-mission/e-mission-docs/issues/950#issuecomment-1803287535

> Note that there is not really "the model" in the interface. Instead, all the interface methods take a key. Right now, we have only one model (the trip model), but we may have a user model or a tour model or other models in the future.